pkg/base/k8s/kubeclient.go

```diff
@@ -345,6 +345,24 @@ func (k *KubeClient) GetNodes(ctx context.Context) ([]coreV1.Node, error) {
 	return nodes.Items, nil
 }
 
+// GetSystemDriveUUID returns system drive uuid
+// Receives golang context
+// Returns return string - drive uuid, error
+func (k *KubeClient) GetSystemDriveUUID(ctx context.Context, node string) string {
```

Comment:
How we handle situation when OS installed on top of RAID and we have multiple system drives?

Reply:
I didn't implement support for multiple system disks. I'll think about it

Reply:
@ykulazhenkov I assume we don't support lvg on multiple system disks yet. As I see we create LVG for one system disk. Does it make sense to support this case first?

Reply:
If I understand you correctly, you talk about this use case:
/dev/sda, /dev/sdb -> LVM system-VG - > LVM root-lv (used for /)- 2 physical devices in VG, LV with root fs in this VG
I think we should support this use case. Drives /dev/sda and /dev/sdb should be marked as system drives.

Originally I was talking about another use cases with non-LVM RAID:
/dev/sda, /dev/sdb -> RAID /dev/md1 -> /  OR
/dev/sda, /dev/sdb -> RAID /dev/md1 -> LVM system-VG -> LVM root-lv (used for /)
We should improve system drives detection to handle this scenarios.

Reply:
Okay, I've got you